### PR TITLE
Add header with title and logout icon

### PR DESCRIPTION
**Pull request by steinberggit**
Preview: [Storybook Header](https://transmitter-header-2iiukvcif7f.herokuapp.com/storybook/?path=/story/header--header
)
Using a map function for a menu icon seems a bit overtop, but I see it as a preparation for the footer menu 😌

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+`;
+
+function Header({ links, active, activeRotate, onItemClick }) {
+  return (
+    <Container>
+      <HeaderTitleSmall>Transmitter</HeaderTitleSmall>
+      {links.map((link) => (
+        <LogoutContainer
+          key={link.label}
+          active={active === link.label}
```
Comment (lmachens) [inline client/src/components/Header.js:43]: `LogoutContainer` doesn't know about `active and `activeRotate`. You can remove both.

File: client/src/assets/icons/Icons.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+export const Rocket = ({ active, activeRotate }) => {
+  return (
+    <SVG
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      width="50.0px"
+      height="50.0px"
+      viewBox="-180 80 710 760"
+      active={active}
+      activeRotate={activeRotate}
```
Comment (lmachens) [inline client/src/assets/icons/Icons.js:22]: the SVG doesn't know about `activeRotate`:

```suggestion
```

File: client/src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          <LogoutText active={active === link.label}>Logout</LogoutText>
+        </LogoutContainer>
+      ))}
+    </Container>
+  );
+}
+
+Header.propTypes = {
+  links: PropTypes.array,
+  active: PropTypes.string,
```
Comment (lmachens) [inline client/src/components/Header.js:60]: `<Header active>` sounds like a boolean. 
I was thinking:
"What is an active header? 
Instead, you want to say which link is active.

```suggestion
  activeLink: PropTypes.string,
```

You might change this again when you add React-Router.

File: client/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  activeRotate: PropTypes.string,
```
Comment (lmachens) [inline client/src/components/Header.js:61]: I don't understand what `activeRotate` is about. I understand, that this property will rotate the logout icon.
What is the different to `active`? 
Is there a use-case where `active` and `activeRotate` is different?

File: client/src/components/Header.js (hunk 1 above)
Comment (steinberggit) [reply client/src/components/Header.js:43]: ✔️

File: client/src/components/Header.js (hunk 3 above)
Comment (steinberggit) [reply client/src/components/Header.js:60]: ✔️

File: client/src/assets/icons/Icons.js (hunk 2 above)
Comment (steinberggit) [reply client/src/assets/icons/Icons.js:22]: ✔️

File: client/src/components/Header.js (hunk 4 above)
Comment (steinberggit) [reply client/src/components/Header.js:61]: ✔️

**Pull request merged**